_includes/docs/user-guide/rule-engine-2-0/ce-external-nodes.md

```diff
@@ -3,6 +3,72 @@ External Nodes used are used to interact with external systems.
 * TOC
 {:toc}
 
+## AWS Lambda Node
+
+<table  style="width:250px;">
+   <thead>
+     <tr>
+	 <td style="text-align: center"><strong><em>Since TB Version 3.7.1</em></strong></td>
+     </tr>
+   </thead>
+</table> 
+
+![image](/images/user-guide/rule-engine-2-0/nodes/external-aws-lambda.png)
+
+Node publish messages to AWS Lambda (Amazon Lambda).
```

Comment:
Node publishes messages to AWS Lambda, a service that lets you run code without provisioning or managing servers. It sends messages using a RequestResponse invocation type. The node uses a pre-configured client and specified function to run.